app/templates/app/analytics.html

```diff
@@ -157,6 +233,10 @@ <h2 class="display h4">The Percentage of the Type of Items</h2>
 <script src="{% static "admin/vendor/chart.js/Chart.min.js" %}"></script>
 <script src="{% static "admin/vendor/jquery-validation/jquery.validate.min.js" %}"></script>
 <script src="{% static "admin/vendor/malihu-custom-scrollbar-plugin/jquery.mCustomScrollbar.concat.min.js" %}"></script>
+<link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/daterangepicker/daterangepicker.css" />
```

Comment:
Please download these files and put in vendor folder. We will be compressing and staticly serving these files.